debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlSchema.java

```diff
@@ -90,8 +90,8 @@ public MySqlSchema(MySqlConnectorConfig configuration,
         super(
                 configuration,
                 topicSelector,
-                TableFilter.fromPredicate(new Filters.Builder(configuration.getConfig()).build().tableFilter()),
-                new Filters.Builder(configuration.getConfig()).build().columnFilter(),
+                TableFilter.fromPredicate(tableFilters.tableFilter()),
```

Comment:
@mtagle I believe this was a bug actually, because it re-recreated the filters instead of taking the ones passed. I think this would have been wrong in the case where this is invoked from `ParallelSnapshotReader`, where the passed filters only represented the "new" tables. WDYT?

Reply:
hm, I agree this looks odd, and wrong. esp. since I'm setting the MySqlSchema filters to the passed in filters further down.